src/officehours_api/models.py

```diff
@@ -206,7 +209,7 @@ def delete(self, *args, **kwargs):
         # Trigger m2m "remove" signals for attendees
         self.attendees.remove(*self.attendees.all())
         self.save()
-        super().delete(*args, **kwargs)
+        return super().delete(*args, **kwargs)
```

Comment:
I fixed a bug where an internal server error was thrown when deleting a queue with meetings. This change fixed it. See https://github.com/makinacorpus/django-safedelete/issues/212 and https://docs.djangoproject.com/en/3.2/ref/models/instances/#deleting-objects

Reply:
Looks good from the change in safedelete and probably good practice to do anyway.